examples/getreadme/main.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
```

Comment:
could you change this to be an inline example, similar to what I did in 32a84d13d2c44f6a093a89da754d8a57a399c90f?